genapi.py

```diff
@@ -908,12 +908,13 @@ def parse_args(argv):
 def setup(ns):
     """Ensure that we are ready to perform code generation. Returns typesystem."""
     # load raw table
-    dbtypes_json = os.path.join(ns.build_dir, 'dbtypes.json')
+    dbtypes_json = os.path.join(ns.src_dir, 'dbtypes.json')
     if not os.path.exists(ns.build_dir):
         os.mkdir(ns.build_dir)
     if not os.path.isfile(dbtypes_json):
-        print('Downloading ' + DBTYPES_JS_URL + ' ...')
-        f = urlopen(DBTYPES_JS_URL)
+        #print('Downloading ' + DBTYPES_JS_URL + ' ...')
```

Comment:
Remove it, please. Also, would you make an issue (if there isn't already
one) covering this? Basically outlining the problem and perhaps proposing a
more long-term solution.

On Fri, Jun 5, 2015 at 1:02 PM, opotowsky <notifications@github.com> wrote:

> In genapi.py
> <https://github.com/cyclus/cymetric/pull/77#discussion_r31836163>:
>
> >      if not os.path.exists(ns.build_dir):
> >          os.mkdir(ns.build_dir)
> >      if not os.path.isfile(dbtypes_json):
> > -        print('Downloading ' + DBTYPES_JS_URL + ' ...')
> > -        f = urlopen(DBTYPES_JS_URL)
> > +        #print('Downloading ' + DBTYPES_JS_URL + ' ...')
>
> @gidden <https://github.com/gidden> to double-check, is this acceptable
> to leave in the commented out code or should I delete it for the PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cymetric/pull/77/files#r31836163>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192